frontend/src/components/status/StatusCheck.jsx

```diff
@@ -41,23 +23,23 @@ class StatusCheck extends React.PureComponent {
     const { description, isPassing } = this.props;
 
     return (
-      <div style={styles.container}>
-        <div style={styles.icon}>
+      <tr>
+        <td>{description}</td>
+        <td>
           {isPassing && (
             <div style={styles.passing}>
-              <Glyphicon style={styles.glyphicon} glyph="ok-circle" />
-              <Label>{LOCALIZE.commons.passStatus}</Label>
+              <i className="fas fa-check-circle" />
+              <span>{LOCALIZE.commons.passStatus}</span>
```

Comment:
Can we add a space between _**< span >**_ and _**{LOCALIZE.commons.passStatus}**_, to have a little space between the check mark and the Pass word like this:

![image](https://user-images.githubusercontent.com/23021242/52787497-04e01780-302c-11e9-94e1-588efadf7d74.png)